packages/table/src/table.tsx

```diff
@@ -781,6 +784,188 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         }
     }
 
+    // Hotkeys
+    // =======
+
+    private maybeRenderCopyHotkey() {
+        const { getCellClipboardData } = this.props;
+        if (getCellClipboardData != null) {
+            return (
+                <Hotkey
+                    key="copy-hotkey"
+                    label="Copy selected table cells"
+                    group="Table"
+                    combo="mod+c"
+                    onKeyDown={this.handleCopy}
+                />
+            );
+        } else {
+            return undefined;
+        }
+    }
+
+    private maybeRenderSelectionResizeHotkeys() {
+        const { allowMultipleSelection, selectionModes } = this.props;
+        const isSomeSelectionModeEnabled = selectionModes.length > 0;
+
+        if (allowMultipleSelection && isSomeSelectionModeEnabled) {
```

Comment:
Any other cases we'd want this feature disabled?

Reply:
Why does allowMultipleSelection need to be necessary? You should be able to resize selection even in single-select modes.

Reply:
...oh, I see, allowMultipleSelection doesn't let you have more than one cell. I thought it was more than one region. Ignore this comment.

Reply:
When `allowMultipleSelection=false`, that actually means only one cell/column/row (or the full table) can be selected at a time:

![2017-10-13 12 04 48](https://user-images.githubusercontent.com/443450/31562054-e5690644-b00e-11e7-96ee-a9054ea1458e.gif)

The prop name is unclear; that's why I want to rename it. See #1625.